sources/packages/api/src/services/notification/gc-notify-actions.service.ts

```diff
@@ -0,0 +1,39 @@
+import { Injectable } from "@nestjs/common";
+import { Student } from "../../database/entities";
+import { getExtendedDateFormat, getPSTPDTDateTime } from "../../utilities";
+import { STUDENT_FILE_UPLOAD_TEMPLATE_ID } from "../../utilities/system-configurations-constants";
+import { GCNotifyResult } from "./gc-notify.model";
+import { GCNotifyService } from "./gc-notify.service";
+
+@Injectable()
+export class GCNotifyActionsService {
+  constructor(private readonly gcNotifyService: GCNotifyService) {}
+
+  /**
+   * This method is used to send email notification to SBC
+   * when a student uploads documents and submits it in the file uploader screen.
+   * @param student
+   * @param documentPurpose
+   * @param applicationNumber
+   * @returns GCNotifyResult.
```

Comment:
I remember while doing the last PR, we thought it is a good idea not to use return object name for @returns. Instead short description.